### PR TITLE
Update roles page

### PR DESCRIPTION
**Pull request by mosabua**
as discussed, see commit messages and preview

### CODE REVIEW

File: development/roles.md [hunk 1]
```diff
@@ -257,26 +257,32 @@ technologies.
```
Comment (martint) [inline development/roles.md:263]: Typo in commit title : "Iimprove"

Comment (mosabua) [reply development/roles.md:263]: fixed

File: development/roles.md [hunk 2]
```diff
@@ -133,6 +133,12 @@ An Apache Hive committer did an excellent write up on their process and much of
 this aligns with our philosophy on maintainers.
 [Read about it](https://cwiki.apache.org/confluence/display/Hive/BecomingACommitter).
 
+Trino maintainers typically focus on the core trino repository, but many also
```
Comment (dain) [inline development/roles.md:136]:
```suggestion
Trino maintainers typically focus on the core Trino repository, but many also
```

File: development/roles.md [hunk 3]
```diff
@@ -257,22 +263,32 @@ technologies.
...
-* [trino-gateway]({{site.github_org_url}}/trino-gateway)
-  * Jaeho Yoo [<i class="fab fa-github"></i> Chaho12](https://github.com/Chaho12)
-  * Star Poon [<i class="fab fa-github"></i> oneonestar](https://github.com/oneonestar)
-  * Vishal Jadhav [<i class="fab fa-github"></i> vishalya](https://github.com/vishalya)
-  * Will Morrison [<i class="fab fa-github"></i> willmostly](https://github.com/willmostly)
-* [trino-go-client]({{site.github_org_url}}/trino-go-client)
-  * Jan Waś [<i class="fab fa-github"></i> nineinchnick](https://github.com/nineinchnick)
-* [trino-js-client]({{site.github_org_url}}/trino-js-client)
-  * Filipe Regadas [<i class="fab fa-github"></i> regadas](https://github.com/regadas)
+[aws-proxy]({{site.github_org_url}}/aws-proxy) - Proxy for AWS S3
```
Comment (dain) [inline development/roles.md:266]: Can we link to the team on github? like this one is https://github.com/orgs/trinodb/teams/maintainers-s3-proxy.  I'm not sure if that is publically visible though.

**Review by dain — APPROVED** [review]

File: development/roles.md (hunk 3 above)
Comment (mosabua) [reply development/roles.md:266]: requires a github account AND maintainer ship I think .. at least I got others to test and they see a 404 ..

File: development/roles.md (hunk 2 above)
Comment (mosabua) [reply development/roles.md:136]: Fixed

**Pull request merged**